packages/v4/patternfly-docs.routes.js

```diff
@@ -9,6 +9,10 @@ module.exports = {
     Component: () => import(/* webpackChunkName: "get-in-touch/index" */ './src/pages/get-in-touch'),
     title: 'Get in touch'
   },
+  '/community': {
+      Component: () => import('./src/pages/community'),
```

Comment:
You should add a webpack chunk name like: `/* webpackChunkName: "community/index" */ `